security.md

```diff
@@ -0,0 +1,39 @@
+# Security
+
+Corteza takes the security of our software products and services seriously,
+which includes all source code repositories managed through our GitHub organizations.
+Here is the list of [Corteza Repositories](https://github.com/orgs/cortezaproject/repositories)
+
+If you believe you have found a security vulnerability in any Corteza repository, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please send email to [security@cortezaproject.org](mailto:security@cortezaproject.org).
+
+If you prefer to submit via forum, please report them to [Corteza Forum](https://forum.cortezaproject.org/).
+
+You should receive a response within 24 business hours. If for some reason you do not,
+please follow up via email to ensure we received your original message. 
+
+Please include the requested information listed below (as much as you can provide)
+to help us better understand the nature and scope of the possible issue:
+
+* Type of issue
+* Full paths of source file(s) related to the manifestation of the issue
+* The location of the affected source code (tag/branch/commit or direct URL)
+* Any special configuration required to reproduce the issue
+* Step-by-step instructions to reproduce the issue
+* Proof-of-concept or exploit code (if possible)
+* Impact of the issue, including how an attacker might exploit the issue
```

Comment:
Have the first bullet start with a capital letter (leave as is), the rest should start with a lower-case letter (change).
All lines except for the last one should end with a comma (,) and the last with a period (.)